zerver/tests/test_subs.py

```diff
@@ -171,20 +172,22 @@ def test_update_stream_admin_subscription(self) -> None:
         stream = get_stream(stream_name, hamlet.realm)
         do_update_stream_admin_subscriptions(stream, stream_admins=[hamlet, self.example_user("prospero")],
                                              value=True)
-        prospero_sub = get_subscription(stream_name, self.example_user("prospero"))
-        self.assertTrue(prospero_sub.is_admin)
+        expected_admins = [hamlet.email, self.example_email("prospero")]
+        self.assertEqual(get_admin_subscriber_emails(stream, hamlet), expected_admins)
```

Comment:
Here, I have updated previous written test to exact match list of stream admins. This way the previous test will be more efficient and new function `get_admin_subscriber_emails` will be test as well.